packages/connect-react/src/components/append/AppendAfterHybridLoginScreen.tsx

```diff
@@ -0,0 +1,104 @@
+import { LockIcon } from '../shared/icons/LockIcon';
+import { PasskeyAddIcon } from '../shared/icons/PasskeyAddIcon';
+import { LinkButton } from '../shared/LinkButton';
+import { Notification } from '../shared/Notification';
+import { PrimaryButton } from '../shared/PrimaryButton';
+
+const AppendAfterHybridLoginScreen = ({ attestationOptions }: { attestationOptions: string }) => {
+  const { config, navigateToScreen } = useAppendProcess();
+  const [dontShowAgain, setDontShowAgain] = useState(false);
+  const [error, setError] = useState<string | undefined>(undefined);
+  const [loading, setLoading] = useState(false);
+  const { getConnectService } = useShared();
+
+  const handleSubmit = useCallback(async () => {
+    if (loading) {
+      return;
+    }
+
+    setLoading(true);
+    setError(undefined);
+
+    const res = await getConnectService().completeAppend(attestationOptions);
+    if (res.err) {
+      log.error('error:', res.val);
+      setLoading(false);
+      setError('Passkey operation was cancelled or timed out.');
+
+      return;
+    }
+
+    if (dontShowAgain) {
+      const createEventRes = await getConnectService().recordEventUserAppendAfterCrossPlatformBlacklisted();
+
+      if (createEventRes?.err) {
+        log.error('error:', createEventRes.val);
+      }
+    }
+
+    setLoading(false);
+    navigateToScreen(AppendScreenType.Success);
+  }, [getConnectService, config, navigateToScreen, loading, attestationOptions]);
+
+  const toggleDontShowAgain = () => setDontShowAgain(prev => !prev);
+
+  const handleSkip = async () => {
+    if (dontShowAgain) {
+      const createEventRes = await getConnectService().recordEventUserAppendAfterCrossPlatformBlacklisted();
+
+      if (createEventRes?.err) {
+        log.error('error:', createEventRes.val);
+      }
+    }
+
+    config.onSkip();
+  };
+
+  return (
+    <div className='cb-append-after-hybrid-login-container'>
+      <div className='cb-h1'>Add a passkey to this device</div>
+      {error ? (
+        <Notification
+          className='cb-error-notification'
+          message={error}
+        />
+      ) : null}
+      <div className='cb-append-after-hybrid-login-icons'>
+        <PasskeyAddIcon className='cb-append-after-hybrid-login-icon' />
+      </div>
+      <div className='cb-append-after-hybrid-login-content'>
+        <LockIcon className='cb-append-after-hybrid-login-lock-icon' />
+        <div className='cb-p'>Add a passkey on this device in order to not require any other devices for login</div>
+      </div>
+      <Checkbox
+        label={"Don't show this again"}
+        checked={dontShowAgain}
+        onChange={toggleDontShowAgain}
+      />
+      <div className='cb-append-after-hybrid-login-cta'>
+        <PrimaryButton
+          isLoading={loading}
+          onClick={() => void handleSubmit()}
+          className='cb-append-after-hybrid-login-button'
+        >
+          Add new passkey
+        </PrimaryButton>
+        <LinkButton
```

Comment:
In the Figma this is left aligned (also our other components are left-aligned at least to my knowledge) => I think we should left-align here as well